src/backend/defs/rest-api/objects/MessageV2.yaml

```diff
@@ -0,0 +1,59 @@
+---
+type: object
+properties:
+  attachments:
+    type: array
+    items:
+      "$ref": Attachment.json
+  body: # in future, could be only a preview of body, limited to xxx chars
```

Comment:
I've no idea of the correct design pattern we will use if any, but yes we will have to extract pertinent informations from whole message to get the valid representation for client visualization.

For example inline attachment will be part of the body